recipes/libssh2/all/conanfile.py

```diff
@@ -82,20 +86,28 @@ def package(self):
         cmake.install()
 
         self.copy("COPYING", dst="licenses", src=self._source_subfolder, keep_path=False)
-        if os.path.exists(os.path.join(self.package_folder, 'lib64')):
+        if os.path.exists(os.path.join(self.package_folder, "lib64")):
             # rhel installs libraries into lib64
-            os.rename(os.path.join(self.package_folder, 'lib64'),
-                      os.path.join(self.package_folder, 'lib'))
+            os.rename(os.path.join(self.package_folder, "lib64"),
+                      os.path.join(self.package_folder, "lib"))
 
-        tools.rmdir(os.path.join(self.package_folder, 'share'))
-        tools.rmdir(os.path.join(self.package_folder, 'lib', 'cmake'))
-        tools.rmdir(os.path.join(self.package_folder, 'lib', 'pkgconfig'))
+        tools.rmdir(os.path.join(self.package_folder, "share"))
+        tools.rmdir(os.path.join(self.package_folder, "lib", "cmake"))
+        tools.rmdir(os.path.join(self.package_folder, "lib", "pkgconfig"))
 
     def package_info(self):
-        lib_name = "libssh2" if self.settings.os == "Windows" else "ssh2"
-        self.cpp_info.libs = [lib_name]
-
+        self.cpp_info.names["cmake_find_package"] = "Libssh2"
+        self.cpp_info.names["cmake_find_package_multi"] = "Libssh2"
+        self.cpp_info.components["_libssh2"].names["cmake_find_package"] = "libssh2"
```

Comment:
dash prefix is only because component name can be the same as package name, right?

Reply:
can't be the same, yes correct ;)